core/rpc/src/rpc_impl.rs

```diff
@@ -162,6 +166,31 @@ pub fn address_to_script(payload: &AddressPayload) -> packed::Script {
     payload.into()
 }
 
+pub fn parse_key_address(addr: &str) -> Result<Address> {
+    if Address::from_str(addr)
+        .map_err(|e| anyhow!("{:?}", e))?
+        .is_secp256k1()
+    {
+        parse_address(addr)
+    } else {
+        Err(MercuryError::rpc(RpcError::KeyAddressIsNotSecp256k1).into())
+    }
+}
+
+pub fn parse_normal_address(addr: &str) -> Result<Address> {
+    Address::from_str(addr).map_err(|e| anyhow!("{:?}", e))
+}
+
+pub fn lock_args_to_sp_key(lock_args: Bytes) -> H160 {
```

Comment:
This function is used to convert pubkey_hash to Secp short address, so I suggest to rename `pubkey_to_secp_address`, which can be used for either acp or cheque.

Reply:
The secp address length is not 20, this is hashed by blake2b160.